lucetc/tests/wasm.rs

```diff
@@ -78,6 +78,32 @@ mod module_data {
         assert_eq!(mdata.function_info().len(), 3);
         assert_eq!(mdata.export_functions()[0].names, vec!["main"]);
     }
+
+    #[test]
+    fn duplicate_imports() {
+        let m = load_wat_module("duplicate_imports");
+        let b = Bindings::from_file(&PathBuf::from(
```

Comment:
should just be able to use `Path::new()` here to avoid allocation, not that it matters hugely in a test